core-java-modules/core-java-lang-2/src/main/java/com/baeldung/methodmultiplereturnvalues/Coordinates.java

```diff
@@ -0,0 +1,62 @@
+package com.baeldung.methodmultiplereturnvalues;
+
+class Coordinates {
+
+    private final double longitude;
+    private final double latitude;
+    private final String directionsNote;
```

Comment:
Let's not make these `final`. It's unnecessary code for this example.

Reply:
I just wanted to make the class immutable. Anyway, I've removed the final modifier from all classes.